plugins/smssync/classes/Controller/Sms/Smssync.php

```diff
@@ -91,7 +98,7 @@ private function _incoming()
 		// Remove Non-Numeric characters because that's what the DB has
 		$to = preg_replace("/[^0-9,.]/", "", $this->request->post('sent_to'));
 
-		$from = preg_replace("/[^0-9,.]/", "", $from);
+		$from = preg_replace("/[^0-9A-Za-z,.]/", "", $from);
```

Comment:
Might be best to make this even more forgiving.. If we can get alpha chars in phone numbers, we can probably get all of unicode letters.